src/main/java/org/skriptlang/skript/bukkit/tags/elements/ExprTag.java

```diff
@@ -59,7 +59,7 @@ public class ExprTag extends SimpleExpression<Tag> {
 	}
 
 	private Expression<String> names;
-	TagType<?>[] types;
+	public TagType<?>[] types;
```

Comment:
why is this public

Reply:
always was public, this just explicitly states it as public

Reply:
It was not public previously, it was package-private

Reply:
When something doesn't have a modifier it means it's accessible to things in the same package level. It's not public to things outside that package.
There's a nice explanation for modifiers [here](<https://www.w3schools.com/java/java_modifiers.asp>) if you'd like to read more about it, since it's quite useful for internal API stuff.

If you want things to be available to extending classes as well, you can use the `protected` modifier.